package Exception_Handling;

public class Use_Of_Throws {
	public static void main(String[] args) throws InterruptedException  {
		Thread.sleep(1000);
		System.out.println("Hi");
		System.out.println(" yes you can do it.");
	}

}
